backend/tests/unit/dal/Services/ProjectServiceTest.cs

```diff
@@ -709,17 +755,27 @@ public void Update_AddBuilding()
             // Act
             var projectToUpdate = service.Get(project.ProjectNumber);
             projectToUpdate.AddProperty(building);
-            var result = service.Update(projectToUpdate);
+            service.Update(projectToUpdate);
+            var result = service.Get(project.ProjectNumber);
 
             // Assert
             Assert.NotNull(result);
-            Assert.Equal(projectToUpdate, result);
             Assert.Single(result.Properties);
+            result.Should().BeEquivalentTo(projectToUpdate, options => options
+            .IgnoringCyclicReferences()
+            .Excluding(x => x.SelectedMemberPath == "Properties[0].Building.Projects")
+            .Excluding(x => x.SelectedMemberPath == "Properties[0].Building.Parcel")
+            .Excluding(x => x.SelectedMemberPath == "Properties[0].Building.Agency.Parcels")
+            .Excluding(x => x.SelectedMemberPath == "Properties[0].Id")
```

Comment:
I do like how expressive this line is, it is essentially saying that I expect the original object to be equal to the database object, except for the Id, which makes sense in an Add workflow.